### ISSUE TITLE
Allow changing preview width with ivy layout

### ISSUE THREAD

**Issue by max397574** [nvim-telescope/telescope.nvim#1503]
**Is your feature request related to a problem? Please describe.**
I can't change the preview width in ivy layout.

**Describe the solution you'd like**
Just be able to change preview width with sth like this:
```
lua require"telescope.builtin".find_files(require"telescope.themes".get_ivy({layout_config={preview_width=0.8}}))
```
When I do this now I get this error:
```
Unsupported layout_config key for the bottom_pane strategy: preview_width                                                                                                                                         
```

**Comment by l-kershaw** [comment]
Hi @max397574,

I agree that this should be an option for the `bottom_pane` layout strategy (hence my marking it as a TODO [here](https://github.com/nvim-telescope/telescope.nvim/blob/ed4adba6d0083a14cabf5837f2511b2617a45593/lua/telescope/pickers/layout_strategies.lua#L813)). However it is not a high priority feature at the moment, so I'm not sure if this will be worked on by any of the core team.

The implementation shouldn't be too difficult, as it can follow the same style as the `preview_width` options for other layouts.
Therefore I am marking this as "good first issue" and will add it to the [Mentored Projects](https://github.com/nvim-telescope/telescope.nvim/issues/1190) list. I am happy to give guidance/feedback on a PR for this, so feel free to ping me if you want to make one.

**Comment by max397574** [comment]
@l-kershaw 
I could just use this right?
https://github.com/nvim-telescope/telescope.nvim/blob/ed4adba6d0083a14cabf5837f2511b2617a45593/lua/telescope/pickers/layout_strategies.lua#L530

**Comment by l-kershaw** [comment]
Yeh, you want something like:

https://github.com/nvim-telescope/telescope.nvim/blob/ed4adba6d0083a14cabf5837f2511b2617a45593/lua/telescope/pickers/layout_strategies.lua#L530-L533

But probably make the default 1/2 rather than 2/3 so that the default behaviour remains the same as current

**Comment by max397574** [comment]
it works but I still get the error

**Comment by l-kershaw** [comment]
What error do you mean? Could you share a screenshot?

**Comment by max397574** [comment]
![Screenshot 2021-11-28 at 12 51 35](https://user-images.githubusercontent.com/81827001/143766493-d13f2805-8cd0-4f53-910c-cf51e2428b48.png)
![Screenshot 2021-11-28 at 12 51 49](https://user-images.githubusercontent.com/81827001/143766502-9c5b9404-ae57-423b-850d-bb8354f34efb.png)

but it works

**Comment by l-kershaw** [comment]
Ah, okay. You need to add something like:
https://github.com/nvim-telescope/telescope.nvim/blob/ed4adba6d0083a14cabf5837f2511b2617a45593/lua/telescope/pickers/layout_strategies.lua#L274
to the area here:
https://github.com/nvim-telescope/telescope.nvim/blob/ed4adba6d0083a14cabf5837f2511b2617a45593/lua/telescope/pickers/layout_strategies.lua#L780-L782

**Comment by max397574** [comment]
https://github.com/nvim-telescope/telescope.nvim/pull/1505
@l-kershaw

**Comment by l-kershaw** [comment]
Closed by #1505
